src/client/tweetAnalysis.js

```diff
@@ -57,15 +57,37 @@ export const groupedCountWords = (countedWords) => {
  * @param tweets
  * @returns {Array}\
  */
-export const mostActiveUsers = (tweets) => {
-  const result = flattenObjectToArray(categoriseByUser(tweets));
-  result.sort(
+export const mostActiveUsers = (tweets) => (
+  categoriseByUser(tweets).sort(
     (tweetList1, tweetList2) => (
-      tweetList1.tweets.length < tweetList2.tweets.length
+      tweetList1.posts.length < tweetList2.posts.length
     )
-  );
-  return result;
-};
+  )
+);
+
+/**
+ * Group a list of posts by their author.
+ * @param [posts]
+ * @returns {{}}
+ */
+const categoriseByUser = (posts) => (
+  posts.reduce((userPostCount, post) => {
+    const categorizedUserIndex = userPostCount.findIndex((categorizedUserPost) => (

```

Comment:
Didn't we come to the conclusion that using .findIndex and babel polyfill was more effort than it was worth? (e.g. seeing as the builds are all failing :( )